my bike images fixed

bita2019 opened the pull request.

Sam-Martin commented on index.html:
```diff
@@ -1,179 +1,144 @@
-            alt="Logo for Bikes for Refugees Scotland"
-            class="header__logo"
-          />
+            <img src="images\bikes-for-refugees_logo.jpg" alt="Logo for Bikes for Refugees Scotland" class="header__logo" />
```
this should be `/` for consistency.

bita2019 replied:
hi Sam, I corrected all that you noted, would you mind to double check them please ? thanks

Sam-Martin commented on index.html:
```diff
@@ -1,179 +1,144 @@
+            </ul>
         </div>
 
         <button>Donate Now</button>
```
Does this look right? Could you post a screenshot of what your page looks like at the moment?

bita2019 replied:
Hi Sam thanks for your comments. I will sort out all .

LucyMac commented:
Hi @bita2019 Can you please edit the name of your PR so that it includes your class and your name please. Look at your classmates's PR names for reference. Thanks 🙂

bita2019 commented:
Hi Lucy
Sorry about any inconvenience.
I will change it.

Sent from my iPhone

> On 15 Nov 2021, at 13:12, Lucy Zidour ***@***.***> wrote:
> 
> ﻿
> Hi @bita2019 Can you please edit the name of your PR so that it includes your class and your name please. Look at your classmates's PR names for reference. Thanks 🙂
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or unsubscribe.

bita2019 commented:
Hi Lucy ,
I have changed my profile name on github , please let me know if I need to do something else.
Changed to : London class8 Maryam Mohsenpour 

Sent from my iPhone

> On 15 Nov 2021, at 13:12, Lucy Zidour ***@***.***> wrote:
> 
> ﻿
> Hi @bita2019 Can you please edit the name of your PR so that it includes your class and your name please. Look at your classmates's PR names for reference. Thanks 🙂
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or unsubscribe.

bita2019 commented:
> Hi @bita2019 Can you please edit the name of your PR so that it includes your class and your name please. Look at your classmates's PR names for reference. Thanks slightly_smiling_face

hi , sorry I don't know how to change my PR name.

DiegoJunior0 commented:
Hi Maryam, this still needs a bit of work:
- Have a look at the spacing between the main menu items
- The link for logo image is not working, can you see why?
- Have a bit more of a go on the size and look of the buttons
- I see you had a try with with the button Hover, have a look at this: https://www.w3schools.com/cssref/sel_hover.asp see if you can get it working
Let me know if you need any help.

bita2019 commented:
Hi Henry,
thanks for your comments.
I will work on it.

On Wed, Nov 17, 2021 at 7:47 PM Henry Skinner ***@***.***>
wrote:

> Hi Maryam, this still needs a bit of work:
>
>    - Have a look at the spacing between the main menu items
>    - The link for logo image is not working, can you see why?
>    - Have a bit more of a go on the size and look of the buttons
>    - I see you had a try with with the button Hover, have a look at this:
>    https://www.w3schools.com/cssref/sel_hover.asp see if you can get it
>    working
>    Let me know if you need any help.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/CodeYourFuture/bikes-for-refugees/pull/162#issuecomment-971917165>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AN53JSXZCPWBJFTQ37BWVATUMQBFVANCNFSM5H6WEJ7A>
> .
>

Sam-Martin commented on index.html:
```diff
@@ -1,179 +1,146 @@
-            <h2 class="heading-underline">Learn more</h2>
+            <div class="hero">
+                <h1 >Bikes for Refugees</h1>
+                <p class="hero h1">
```
I think your hero image styling problems are coming from these classes in the <p> tag!
